src/core/service/notifications/jobs/handler.go

```diff
@@ -102,3 +104,17 @@ func (h *Handler) HandleReplicationTask() {
 		return
 	}
 }
+
+// HandleWebhookJob handles the hook of webhook job
+func (h *Handler) HandleWebhookJob() {
```

Comment:
No need to parse the job status change object?

Reply:
> No need to parse the job status change object?

can't understand the question. would U explain it in detail?

Reply:
> No need to parse the job status change object?

Prepare() method (already exist)will do this